### DISCUSSION TITLE
Import data from another Hosting to Self Hosted Imposter

### DISCUSSION THREAD

**Discussion by Fluffy728** [Impostor/Impostor#574] [Q&A] [answered]
Is it possible and how do I migrate my previous info to new Imposter. I did not set up yet becuase I need to migrate from ServerFlex cloud host to Impostor self host, and serverflex uses Impsoter.

I confirm

- [x] I have searched the internet
- [x] This is not a duplicate
- [x] I checked the wiki probelly
- [x] I confirm that I may get in trouble if this is wrong
- [x] This is a real issue

**Comment by Fluffy728** [comment]
Oh released if I cancel my subscription I can get an option to download my data. But before that will it work and how to import

**Reply by Fluffy728** [reply]
https://serverflex.io/company/legals-and-statements/data-retention-policy

**Reply by miniduikboot** [reply]
We're not affiliated with serverflex and don't support their service. Please contact their support if you need help with their service.

**Comment by Fluffy728** [comment, marked as answer]
Oh, I released they said
- Modify config.json to your liking.
- 
so can put the config file there and we can migrate? Oh I fergot about that I knew that before, so it is not revalant

**Comment by Fluffy728** [comment]
“We're not affiliated with serverflex and don't support their service. Please contact their support if you need help with their service.” 

“ Oh, I released they said

Modify config.json to your liking.
so can put the config file there and we can migrate? Oh I fergot about that I knew that before, so it is not revalant”
